lib/masamune/schema/column.rb

```diff
@@ -243,7 +243,13 @@ def ruby_value(value, recursive = true)
         when Date, DateTime
           value.to_time
         when String
-          Time.parse(value.to_s)
+          if value =~ /^\d+$/
```

Comment:
not necessarily - a YAML field in a CSV encoded file typically spans multiple lines